config/webpack.common.js

```diff
@@ -100,37 +92,13 @@ module.exports = function (options) {
      */
     resolve: {
 
-      alias: {
```

Comment:
why was this removed?

Reply:
The alias was removed because now webpack loads nested node_module folders allowing libs to have their own dependencies that fabric8-ui doesn't need to include directly itself. This is one of the major reasons for the change. The alias was put in as a work around until this PR gets in.